src/bacnet/basic/object/csv.c

```diff
@@ -164,35 +185,100 @@ bool CharacterString_Value_Valid_Instance(uint32_t object_instance)
     unsigned index = 0; /* offset from instance lookup */
 
     index = CharacterString_Value_Instance_To_Index(object_instance);
-    if (index < MAX_CHARACTERSTRING_VALUES) {
-        return true;
+    for (index = 0; index < CSV_Max_Index; index++) {
+        if (object_instance == CSV_Descr[index].Instance) 
+        {
```

Comment:
put the { in the if statement line for consistency.

Reply:
have it formatted